utils/auth_jwt.ts

```diff
@@ -62,3 +69,36 @@ export const getValidateAuthJWT = ({
         TE.filterOrElse(AuthJWT.is, () => E.toError("Invalid AuthJWT payload"))
       )
   );
+
+/**
+ * A middleware that verify the jwt
+ *
+ * @param jwtConfig the config for the jwt
+ * */
+
+export const verifyJWTMiddleware = (
+  jwtConfig: JWTConfig
+): IRequestMiddleware<"IResponseErrorValidation", AuthJWT> => (
```

Comment:
`AuthJWT` should be configurable and not hardcoded in the method signature

Reply:
This would need a refactor of the `getValidateAuthJWT`

Reply:
We'll make this change on another PR. Thanks.